appcues/src/main/java/com/appcues/ui/extensions/StyleComponentExt.kt

```diff
@@ -56,11 +64,27 @@ internal fun ComponentStyle.getTextAlignment(): TextAlign? {
     }
 }
 
-internal fun ComponentStyle.getFontFamily(context: Context): FontFamily? {
-    return FontFamily.getSystemFontFamily(fontName)
-        ?: FontFamily.getFontResource(context, fontName)
+@Composable
+internal fun ComponentStyle.getFontFamily(): FontFamily? {
+    val context = LocalContext.current
+    val packageNames = LocalPackageNames.current
+    val fontFamily = FontFamily.getSystemFontFamily(fontName)
+        ?: FontFamily.getFontResource(context, packageNames, fontName)
         ?: FontFamily.getFontAsset(context, fontName)
         ?: FontFamily.getSystemFont(fontName)
+
+    val logcues = LocalLogcues.current
+    LaunchedEffect(Unit) {
+        if (fontName != null && fontFamily == null) {
+            logcues.error(
```

Comment:
I guess we don't have warnings, but this feels more like a warning, since things can still run with fallback fonts right?

Reply:
makes sense